tavern/schemas/tests.schema.yaml

```diff
@@ -156,8 +156,18 @@ schema;stage:
           required: false
           func: validate_timeout_tuple_or_float
 
+        cert:
+          func: validate_extensions
+          type: seq
```

Comment:
This will never load extension functions so using `validate_extensions` is wrong here - this can be either a tuple or a string, a extra extension function in `extensions.py` will be required to validate it (there are lots of examples of similar functions in that file)